cmd/generate/config/rules/atlassian.go

```diff
@@ -11,15 +11,23 @@ func Atlassian() *config.Rule {
 	r := config.Rule{
 		Description: "Detected an Atlassian API token, posing a threat to project management and collaboration tool security and data confidentiality.",
 		RuleID:      "atlassian-api-token",
-		Regex: utils.GenerateSemiGenericRegex([]string{
-			"atlassian", "confluence", "jira"}, utils.AlphaNumeric("24"), true),
+		Regex: utils.MergeRegexps(
+			utils.GenerateSemiGenericRegex(
+				[]string{"atlassian", "confluence", "jira"},
+				utils.AlphaNumeric("24"),
+				true,
```

Comment:
Testing this a bit more, I think the first one needs to be changed as follows:
```suggestion
				`[a-zA-Z0-9{24}`,
				false,
```

Otherwise, the `(?i)` at the start of the pattern also makes `ATATT3` case-insensitive. I would have thought `(?:(?i)a|b)` would only apply to `a` but it applies to both.

Reply:
I didn't know that. I will change it